### PR TITLE
Add message if no results show up

### PR DESCRIPTION
**Pull request by kpancha**

### CODE REVIEW

File: project/frontend/src/FindMatch/FindMatch.js [hunk 1]
```diff
@@ -165,6 +165,16 @@ class PreferenceForm extends React.Component {
           restaurants = restaurants.concat(restaurant.results);
         }
 
+        if (restaurants.length === 0) {
+          let message = "We can't find any restaurants near you";
+          if (this.state.open) {
+            message += ' that are currently open';
+          }
+          message += '. Please try changing your preferences.';
```
Comment (surbhimadan1995) [inline project/frontend/src/FindMatch/FindMatch.js:173]: nit: lets make this 
+= ' that match your preferences. Please try changing your preferences' 
(im just being pedantic about adding context to the cant find restaurants near you 🙃  )

File: project/frontend/src/FindMatch/FindMatch.js [hunk 2]
```diff
@@ -165,6 +165,16 @@ class PreferenceForm extends React.Component {
...
+          alert(message);
```
Comment (surbhimadan1995) [inline project/frontend/src/FindMatch/FindMatch.js:174]: possibly for a follow up- but I still think a pop-up alert could be too jarring, maybe just displaying this message at the place where we would show the results if there were any?

File: project/backend/src/main/java/com/google/sps/data/RestaurantScorer.java [hunk 3]
```diff
@@ -115,7 +115,7 @@ private static long roundMetersToMiles(double numMeters) {
   /** Skews ratings so that ratings with a lower number of ratings are weighed less. */
   private static double calculateRatingScore(
       double avgRating, int numRatings, DescriptiveStatistics statistics) {
-    int numInitialRatings = (int) Math.round(statistics.getPercentile(25));
+    int numInitialRatings = (int) Math.round(statistics.getPercentile(10));
```
Comment (surbhimadan1995) [inline project/backend/src/main/java/com/google/sps/data/RestaurantScorer.java:118]: is this change related ?

File: project/frontend/src/FindMatch/FindMatch.js (hunk 2 above)
Comment (jesseelizabeth) [reply project/frontend/src/FindMatch/FindMatch.js:174]: you could just create a NoResults component that would be rendered in this case

File: project/backend/src/main/java/com/google/sps/data/RestaurantScorer.java (hunk 3 above)
Comment (kpancha) [reply project/backend/src/main/java/com/google/sps/data/RestaurantScorer.java:118]: Oh no it's not, probably should've put this in a different PR

File: project/frontend/src/scripts/recommendation_script.js [hunk 4]
```diff
@@ -8,6 +8,17 @@ export default function getRecommendation(preferences, callback) {
     )
     .then((data) => {
       let restaurants = [];
+      if (restaurants.length === 0) {
+        let message = "We can't find any restaurants near you";
+        if (this.state.open) {
+          message += ' that are currently open';
+        }
+        message +=
+          ' and match your preferences. Please try changing your preferences.';
```
Comment (surbhimadan1995) [inline project/frontend/src/scripts/recommendation_script.js:17]: nit: "that" instead of "and"
